src/apiMocks/mockData.ts

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { faker } from "@faker-js/faker";
+import { ScanResult } from "../app/functions/server/onerep";
+import { StateAbbr } from "../utils/states";
+
+// Setting this to a constant value produces the same result when the same methods
+// with the same version of faker are called.
+// @see https://faker.readthedocs.io/en/master/index.html#seeding-the-generator
+const fakerSeed = 123;
+
+// This is a full list of scan results, all pages, and would normally be
+// stored in the `onerep_scan_results.onerep_scan_results` column
+// as `jsonb`.
+export function mockedOneRepScanResults() {
+  faker.seed(fakerSeed);
+  return {
+    data: Array.from({ length: 3 }, () => createRandomScan()),
+  };
+}
+
+/**
+ * Generates scan result with randomly-generated mock data.
+ *
+ * @returns {ScanResult} - A single scan result.
+ */
+export function createRandomScan(): ScanResult {
+  return {
+    id: faker.number.int(),
+    profile_id: faker.number.int(),
+    first_name: faker.person.firstName(),
+    last_name: faker.person.lastName(),
+    middle_name: faker.person.middleName(),
+    age: faker.number.int({ min: 14, max: 120 }).toString(),
+    status: "new", // state is always "new" by default, see the `ScanResult` type definition.
+    addresses: Array.from({ length: 3 }, () => ({
+      zip: faker.location.zipCode(),
+      city: faker.location.city(),
+      state: faker.location.state({ abbreviated: true }) as StateAbbr,
+      street: faker.location.street(),
+    })),
```

Comment:
Interesting, I just noticed that addresses don't have countries. Do we think this feature will ever extend outside of the USA?

Reply:
No plans at this time, they would need to extend their API (likely by adding a new field, but no reason to worry yet :))